Assets/Scripts/UI/LogText/Weather.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Fumbbl.Model.Types;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Fumbbl.UI.LogText
+{
+    public class Weather : LogTextGenerator<Ffb.Dto.Reports.Weather>
+    {
+
+        public override IEnumerable<LogRecord> Convert(Ffb.Dto.Reports.Weather report)
+        {
+         
+            yield return new LogRecord($"<b>Weather Roll {WeatherRollString(report.weatherRoll)}</b>");
+            yield return new LogRecord($"Weather is {report.weather}", 1);
+        }
+
+
+        private string WeatherRollString(int[] rolls)
```

Comment:
This is the exact same method as earlier in this PR. Refactor this into shared code; can be placed in the LogTextGenerator superclass, or possibly in some utility static class.